schemas/telemetry/focus-event/focus-event.1.schema.json

```diff
@@ -24,11 +24,11 @@
             "type": "string"
           },
           {
-            "type": "string"
+            "type": [ "string", "null" ]
```

Comment:
Yes, see the spec here: http://gecko.readthedocs.io/en/latest/toolkit/components/telemetry/telemetry/collection/events.html#eventtelemetry